resources/vocabulary.md

```diff
@@ -93,8 +93,18 @@ sum_of_nums_under(4) // 10
 sum_of_nums_under(6) // 21
 ```
 
+## Process and Tools
```

Comment:
Kind of unrelated, but do you want to change the two main sections of this page to ### Things most programming languages do  and ### Process and tools (like delete line 3)? The outline right now is a little weird.